libs/blocks/merch/merch.js

```diff
@@ -478,9 +478,7 @@ async function openExternalModal(url, getModal, extraOptions) {
 
 const isInternalModal = (url) => /\/fragments\//.test(url);
 
-export async function openModal(e, url, offerType, hash, extraOptions) {
-  e.preventDefault();
-  e.stopImmediatePropagation();
+export async function openModal(url, offerType, hash, extraOptions) {
```

Comment:
unrelated change?
are you sure these are not required?
```
e.preventDefault();
  e.stopImmediatePropagation();
```

Reply:
still to be on the safe side, I reverted this change, because when href is set, it actually follows the link.